app/controllers/requests_controller.rb

```diff
@@ -17,7 +17,7 @@ def show
   # pre-filled distribution containing all the requested items.
   def fullfill
     request = Request.find(params[:id])
-    request.update(status: "Fulfilled")
+    request.update(status: "New Distribution for this Request")
```

Comment:
Looking at the `STATUSES` array you have in `/app/models/request.rb` -- I don't see "New Distribution for this Request" in that list -- are you sure you didn't mean to use a different status here?

It looks like the original issue suggests using this string as the text for a button -- was that what you meant to change?

Reply:
You know, that sounds more like what we needed to do, when we did this issue we thought all that was asked was to change the name of the button "Fulfilled" to "New Distribution for this Request". When we checked in running the files, it didn't have any issues and the button name was changed so we thought it was finished. Should we have made a new button for "New Distribution for this Request" instead?
![Screen Shot 2019-04-23 at 3 17 07 PM](https://user-images.githubusercontent.com/23445639/56612813-e58cda00-65da-11e9-9a8b-e816bd22b902.png)
![Screen Shot 2019-04-23 at 3 16 54 PM](https://user-images.githubusercontent.com/23445639/56612815-e58cda00-65da-11e9-8402-c9e88cb22f68.png)

These are screens of what those look like with the code included here.